src/ros.cpp

```diff
@@ -98,9 +98,12 @@ void RosPosePublisher::setFrame(QString frame)
 void RosPosePublisher::publish(){
     double x,y, theta;
     if (_origin) {
-        x = _origin->mapToItem(_target,QPoint(0,0)).x() * _pixel2meter;
-        y = _origin->mapToItem(_target,QPoint(0,0)).y() * _pixel2meter;
+        x = _origin->mapFromItem(_target,QPoint(0,0)).x() * _pixel2meter;
+        y = - _origin->mapFromItem(_target,QPoint(0,0)).y() * _pixel2meter;
```

Comment:
This does not belong to this commit (I believe it should be part of the previous commit)